dna/src/lib.rs

```diff
@@ -132,11 +132,11 @@ impl Dna {
     /// use holochain_dna::Dna;
     ///
     /// let dna = Dna::new();
-    /// println!("json: {}", dna.to_json().unwrap());
+    /// println!("json: {}", dna.to_json());
     ///
     /// ```
-    pub fn to_json(&self) -> serde_json::Result<String> {
-        serde_json::to_string(self)
+    pub fn to_json(&self) -> String {
+        serde_json::to_string(self).expect("DNA should serialize")
```

Comment:
Everywhere I looked called `unwrap` on the result that this returns. This is one of the few situations where panicking does make sense because it's supposed to be guaranteed that it can serialize and it failing to is unrecoverable.